src/boards/include/boards/adafruit_fruit_jam.h

```diff
@@ -28,6 +28,67 @@
 // This means RP2350B.
 #define PICO_RP2350A 0
 
+// --- BOARD SPECIFIC ---
+// On JST PH connector.
+#define ADAFRUIT_FRUIT_JAM_A0_PIN 40
+
+#define ADAFRUIT_FRUIT_JAM_A1_PIN 41
```

Comment:
Is the blank line here deliberate?

Reply:
Yes, the A0 pin is on a separate header from the other An pins.